packages/core/src/testing/renderWeb3Hook.tsx

```diff
@@ -8,6 +8,7 @@ import { ReadonlyNetworksProvider } from '../providers/network'
 
 export interface renderWeb3HookOptions<Tprops> {
   mockProvider?: MockProvider
+  otherProvider?: MockProvider
```

Comment:
I would make it more generic:

```typescript
mockProvider?: MockProvider | Record<number /* ChainId */, MockProvider>
```

so the provider passed in is either a single provider or a map of `chainId => provider` if you want multiple.